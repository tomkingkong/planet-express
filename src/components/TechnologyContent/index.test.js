import React from 'react';
import { shallow } from 'enzyme';

import { TechnologyContent } from '.';

describe('TechnologyContent component', () => {
  it('should match snapshot', () => {
    const mockProps = {
      org: '', 
      title: '', 
      status: '',
      benefits: [], 
      techAreas: ['words, bro'], 
      description: '' 
    }
    const wrapper = shallow(
    <TechnologyContent 
      {...mockProps}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});