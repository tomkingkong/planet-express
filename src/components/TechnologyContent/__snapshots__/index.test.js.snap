// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechnologyContent component should match snapshot 1`] = `
<article
  className="Technology"
>
  <h4
    className="tech__title"
  />
  <p
    className="tech__desc"
  />
  <p
    className="tech__org"
  >
    Organization: 
  </p>
  <p
    className="tech__status"
  >
    Status: 
  </p>
  <ul
    className="tech__techTags"
  >
    <li
      className="tech__tag"
      key="words0"
    >
      words
    </li>
    <li
      className="tech__tag"
      key=" bro1"
    >
       bro
    </li>
  </ul>
</article>
`;
